<launch>	
	 <node name="qbo_face_tracking" pkg="qbo_face_tracking" type="qbo_face_tracking" output="screen" >

		 <!-- <param name="face_classifier_path" value="0" /> -->
                 <param name="alternative_face_classifier_path" value="/usr/share/OpenCV-2.3.1/haarcascades/haarcascade_profileface.xml" />
		 <!-- <param name="default_pos_x" value="0" /> -->
	 	<!-- <param name="default_pos_y" value="10" /> -->
		 <param name="check_Haar" value="1" /> 
		 <!-- <param name="check_track_object" value="50" /> -->
		 <!-- <param name="undetected_threshold" value="5.0" /> -->
		 <!-- <param name="distance_threshold" value="5.0" /> -->
	 <!-- <param name="send_to_recognizer" value="false" /> -->	
	 </node>
</launch>

